src/components/TransactionContractComponents/index.tsx

```diff
@@ -447,41 +447,37 @@ export const CreateAsset: React.FC<PropsWithChildren<IIndexedContract>> = ({
                 </Link>
                 <Copy data={parameter.royalties?.address}></Copy>
               </CenteredRow>
-              {parameter.royalties?.transferFixed && (
+              <CenteredRow>
                 <span>
                   <strong>Transfer Fixed:&nbsp;</strong>
-                  {parameter?.royalties?.transferFixed / 1000000} KLV
+                  {parameter?.royalties?.transferFixed / 1000000 || 0} KLV
```

Comment:
use pattern 10 ** 6 which is easier to read

Reply:
Done!